### DISCUSSION TITLE
.st and .rawtlt

### DISCUSSION THREAD

**Discussion by stav-ros** [nextpyp/.github#116] [Q&A] [answered]
Is there a way to import .st and .rawtlt files directly? Or at least a way to convert them appropriately for nextpyp?

**Comment by abartesaghi** [comment, marked as answer]
Not right now, unfortunately. This will supported in the next release of nextpyp though. In most cases, it will be easier to re-align the tilt-series from scratch within nextpyp. Alternatively, you could use the pre-aligned .st files as input raw data, but that will not be ideal because the .st files will already have one round interpolation applied to them.
